src/ui/public/notify/partials/toaster.html

```diff
@@ -7,7 +7,9 @@
 
         <i class="fa" ng-class="'fa-' + notif.icon" tooltip="{{notif.title}}"></i>
 
-        <kbn-truncated source="{{notif.content | markdown}}" is-html="true" length="{{notif.truncationLength}}" class="toast-message" /></kbn-truncated>
+        <kbn-truncated ng-if="notif.content" source="{{notif.content | markdown}}" is-html="true" length="{{notif.truncationLength}}" class="toast-message" /></kbn-truncated>
```

Comment:
Sure, but I think the opening tag shouldn't have a slash before the closing `>`. Looks like a typo in the original.